zipkin-server/README.md

```diff
@@ -268,6 +268,13 @@ The following apply when `STORAGE_TYPE` is set to `elasticsearch`:
                            to 0 as it would mean a machine failure results in data loss.
     * `ES_USERNAME` and `ES_PASSWORD`: Elasticsearch basic authentication, which defaults to empty string.
                                        Use when X-Pack security (formerly Shield) is in place.
+    * `ES_SECURITY_FILE_PATH`: Elasticsearch security properties file path, which defaults to empty
+                               string. The suffix of file should be `.properties`. The keys of
+                               `username` and `password` are `zipkin.storage.elasticsearch.username`
+                               and `zipkin.storage.elasticsearch.password` respectively.
+                               Before the credentials are loaded from the file, `ES_USERNAME` and
+                               `ES_PASSWORD` is in place. If they are empty, basic authentication
+                               is disable temporarily.
```

Comment:
The main feature of this file is that it's reloaded periodically , can we mention that here ?

Reply:
Add the "periodically load"